quiz/src/main/java/com/epam/spring/homework3/quiz/controller/QuestionController.java

```diff
@@ -0,0 +1,77 @@
+package com.epam.spring.homework3.quiz.controller;
+
+import com.epam.spring.homework3.quiz.controller.assembler.QuestionAssembler;
+import com.epam.spring.homework3.quiz.controller.assembler.model.QuestionModel;
+import com.epam.spring.homework3.quiz.controller.dto.QuestionDto;
+import com.epam.spring.homework3.quiz.controller.dto.group.OnCreate;
+import com.epam.spring.homework3.quiz.controller.dto.group.OnUpdate;
+import com.epam.spring.homework3.quiz.controller.mapper.QuestionMapper;
+import com.epam.spring.homework3.quiz.service.QuestionService;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Slf4j
+@RestController
+@RequestMapping("/question")
+@RequiredArgsConstructor
+@Api(tags = "QuestionController description for SWAGGER documentation")
+public class QuestionController {
```

Comment:
same comment  about URL naming as for `UserController`
And also interesting one - update it:

`getAllQuestionsByParentQuizId` -> GET /quiz/{quizId}/questions